lib/config/dry_validation_requirements.rb

```diff
@@ -1,5 +1,5 @@
-# frozen_string_literal: true
+require_relative 'validation/requirements'
 
 module Config
-  DRY_VALIDATION_REQUIREMENTS = ['~> 1.0', '>= 1.0.0'].freeze
+  include ValidationRequirements
```

Comment:
Why did you have to do that ?

This is far from respecting ruby's conventions!

Maybe 
```suggestion
  def self.dry_validation_requirements
    ['~> 1.0', '>= 1.0.0'].map { |s| Gem::Requirement.new s }
  end
```

as you suggested?

Reply:
Is not working, it's like `Config` is unloaded at some point and reinitialised and `requires` are not gonna reload because they were already loaded.

Reply:
In that case I'd rather keep the current version, it is more likely to be accepted by owners, and we'll get a chance to fix the tests (which are the source of error!) if they want the patch.

Worst case scenario, you could replace the `require` by a `load` in `lib/config/validation/schema.rb` explaining that this is due to test failure :)

Anyway, I'm strongly against pushing such change, it is a great example of [test-induced ~design~ damage](https://dhh.dk/2014/test-induced-design-damage.html) IMHO

Reply:
`load` made the trick 🎉